pom.xml

```diff
@@ -78,7 +78,7 @@
     </scm>
 
     <properties>
-        <java.version>11</java.version>
+        <java.version>17</java.version>
```

Comment:
Good change, we missed the news that Java 11 ended premier support last year.

Reply:
Thanks, an I' working on updage gradle to 8.x with spring boot 3 java 17